ang/volunteer/VolOppsCtrl.html

```diff
@@ -16,7 +16,12 @@
   </div>
 
   <!-- Search Form -->
-  <div class="crm-vol-half-responsive crm-vol-opp-search">
+  <div class="crm-vol-half-responsive crm-vol-opp-search" ng-show="allowShowSearch">
```

Comment:
To ensure we don't show the button when the form is already displayed, how about we do:

`ng-show="allowShowSearch && hideSearch"`?

(My simplification of the logic in the controller requires this change.)